NuGet.config

```diff
@@ -19,11 +19,18 @@
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
     <add key="orleans-nightly" value="https://orleans.pkgs.visualstudio.com/orleans-public/_packaging/orleans-nightly/nuget/v3/index.json" />
     <add key="dotnet9-transport" value="https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet9-transport/nuget/v3/index.json" />
+    <add key="azure-sdk-for-net-dev" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" />
```

Comment:
When will these be available on nuget.org? Doing this isn't keeping us in a "shippable state" and we will need to remove this feed and update the versions before we release preview6.

Reply:
This will be removed before the PR merges. Working through this with the Azure SDK team. This was just to get it working and validated so we can greenlight that.